cli/internal/config/config_file.go

```diff
@@ -57,6 +58,11 @@ func (p FlagSet) GetToken() (string, error) {
 	return p.GetString("token")
 }
 
+// GetCwd returns the value of the `cwd` flag. Used to implement CLIConfigProvider interface.
+func (p FlagSet) GetCwd() (string, error) {
```

Comment:
With the port finished do we still need this interface and impl?

Reply:
No, I could tear all of those out and probably do a bunch of renaming. I didn't want to make this PR even larger though. If you want, I can do that.

Reply:
It's okay not to do in this PR. Just wanted to make sure I'm not misunderstanding the use of this interface.